android/anc/build.gradle

```diff
@@ -151,4 +153,7 @@ dependencies {
     androidTestImplementation deps.atsl.ext_junit
     androidTestImplementation deps.atsl.espresso
     implementation deps.work.runtime
+
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4:1.0.1")
```

Comment:
Currently on Unit tests supported on CI. BDD Gherkin tests are being implemented separately for ui/integration

Reply:
done